### ISSUE TITLE
All the time get error with restart of HA [Bug] 

### ISSUE THREAD

**Issue by smarthomefamilyverrips** [Vaskivskyi/ha-asusrouter#549]
### The problem

@Vaskivskyi every time I restart HA I get the following error message in my logs

This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/asusrouter/config_flow.py:548
Integration: AsusRouter (documentation, issues)
First occurred: 10:07:36 (1 occurrences)
Last logged: 10:07:36

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/discovery_flow.py", line 71, in _async_start
    await gather_with_concurrency(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 201, in gather_with_concurrency
    return await gather(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 199, in sem_task
    return await task
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 798, in async_init
    flow, result = await task
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 826, in _async_init
    result = await self._async_handle_step(flow, flow.init_step, data)
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 335, in _async_handle_step
    result: FlowResult = await getattr(flow, method)(user_input)
  File "/config/custom_components/asusrouter/config_flow.py", line 548, in async_step_ssdp
    discovery_info.upnp[ssdp.ATTR_UPNP_MODEL_DESCRIPTION]
KeyError: 'modelDescription'

### Your device model

ASUS XT-12

### Firmware type

Stock

### Firmware version

3.0.0.4.388_22127

### Integration version with the issue

since 0.21.0 now am on 0.21.1

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

2023.2.5

### Is there anything useful in the logs?

```txt
This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/asusrouter/config_flow.py:548
Integration: AsusRouter (documentation, issues)
First occurred: 10:07:36 (1 occurrences)
Last logged: 10:07:36

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/discovery_flow.py", line 71, in _async_start
    await gather_with_concurrency(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 201, in gather_with_concurrency
    return await gather(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 199, in sem_task
    return await task
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 798, in async_init
    flow, result = await task
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 826, in _async_init
    result = await self._async_handle_step(flow, flow.init_step, data)
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 335, in _async_handle_step
    result: FlowResult = await getattr(flow, method)(user_input)
  File "/config/custom_components/asusrouter/config_flow.py", line 548, in async_step_ssdp
    discovery_info.upnp[ssdp.ATTR_UPNP_MODEL_DESCRIPTION]
KeyError: 'modelDescription'
```

### Diagnostics information

Can not attach from mobile phone

### Additional information

None

**Comment by Vaskivskyi** [comment]
Hello, @smarthomefamilyverrips,

Sorry to hear, you are experiencing issues with AsusRouter

From the log error I see, that your device is not reporting the model name in the network discovery (which was added in 0.21.0)

The fix will be available as soon as possible. I will let you know

**Comment by smarthomefamilyverrips** [comment]
@Vaskivskyi, OK happy you see the problem, will await for your fix, Thanks! 👍🏻

**Comment by Vaskivskyi** [comment]
Hey. The issue will be fixed in the next AsusRouter update

**Comment by smarthomefamilyverrips** [comment]
@Vaskivskyi Great thanks! 👍🏻

**Comment by Vaskivskyi** [comment]
Hello. The issue should be fixed in 0.22.0, which is already available via HACS

**Comment by smarthomefamilyverrips** [comment]
@Vaskivskyi did fix it for me! 👍🏻
